pkgs/by-name/is/isisdl/package.nix

```diff
@@ -0,0 +1,65 @@
+{ lib
+, fetchPypi
+, python3Packages
+, util-linux
+, withFFmpeg ? false, ffmpeg
```

Comment:
is `FFmpeg` such a big dependency that we need a feature flag and a separate package ( `isisdl-ffmpeg`)?

Reply:
probably not. it's just a runtime dependency. i've removed it.